src/DataStore.ts

```diff
@@ -36,7 +36,8 @@ export interface JoinConfig {
 /**
  * Sends a voice state update to the main websocket shard of a guild, to indicate joining/leaving/moving across
  * voice channels.
- * @param voiceChannel The voice channel to move to
+ *
+ * @param voiceChannel - The voice channel to move to
```

Comment:
```suggestion
 * @param config - The configuration to use when joining the voice channel
```

Not an issue created by your PR, but I thought it might as well be fixed here!